internal/certs/certs.go

```diff
@@ -14,15 +14,17 @@ import (
 )
 
 const (
-	bitSize = 4096
-	base    = 2
-	exp     = 159
+	bitSize      = 2048
```

Comment:
2048 bits are not enough for a RSA key nowadays. In fact, looking at the `sprig` library they use a 4096 key (see [here](https://github.com/Masterminds/sprig/blob/e708470d529a10ac1a3f02ab6fdd339b65958372/crypto.go#L158-L159))

Reply:
I was wrong, both with the link and with the statement about RSA 2048 not being enough.

Looking at the latest (July 2024) [NIST publication on the topic](https://csrc.nist.gov/pubs/sp/800/78/5/final) this is fine until 2030:

![image](https://github.com/user-attachments/assets/3df04ad2-0af7-4ede-8c03-abe8bc3045a6)